diagnostic.md

```diff
@@ -8,6 +8,12 @@ Assume all code executes in strict mode.
 ## Question 1
 
 Please list the JavaScript primitives and give one example of each.
+Not a number
+Number
+String
+Boolean
+Null
+Undefined
```

Comment:
Believe it or not `NaN` is technically a `number` in javascript as silly as that sounds. So there are only 5 primitives, we also wanted an example of each one.